apache/superset
# fix: change database database selector

### SUMMARY
This pr fixes and issue where the database selector show an error toast when changing database. The issue come about when the params are undefined causing a method to error out.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
before

https://user-images.githubusercontent.com/17326228/152904976-bc74befd-0ff3-46ad-88a1-fa9e200bc371.mp4

after

https://user-images.githubusercontent.com/17326228/152906420-2c0ff7ab-f38c-4811-8a00-82a7d7ab9ea6.mov



### TESTING INSTRUCTIONS
Go to sqlab and click on database  selector. It should not give you an error toast when changing database.

### ADDITIONAL INFORMATION
